ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -187,10 +187,13 @@ class ActionBar extends React.Component {
         staticActionVariables: results.staticActionVariables,
       });
 
-      this.thNotify.send(
-        `Request sent to create an interactive job via actions.json.
-          You will soon receive an email containing a link to interact with the task.`,
-        'success');
+      this.thNotify.send('Request sent to create an interactive job via actions.json.', 'success');
+
+      Object.assign(window.open(), {
+        opener: null,
+        location: getInteractiveTaskUrl(taskId),
+        target: '_blank',
+      });
```

Comment:
Is this to the right task?  I think this will take you to the action task, rather than the interactive task.

Reply:
Seems to work. Do you get the wrong task?

Reply:
OK, I don't understand how it works (since the interactive task isn't created until the action task runs) but you can't argue with success :)